src/core/github.js

```diff
@@ -9,6 +9,44 @@ import { pub } from "core/pubsubhub";
 
 export const name = "core/github";
 
+var body = document.body;
+var head = document.head;
+var issueBase;
+
+var fileIssueLink = document.createElement('button');
+fileIssueLink.setAttribute('data-balloon','file an issue related to the text selected');
+fileIssueLink.setAttribute('data-balloon-pos','left');
+fileIssueLink.classList.add('removeOnSave');
+fileIssueLink.id = 'respec-file-issue-btn';
+fileIssueLink.innerHTML = 'File Issue';
+fileIssueLink.onclick = function () {
+  window.location.href = fileIssueLink.getAttribute('data-href');
+  fileIssueLink.style.display = 'none';
+};
+
+var link = document.createElement('link');
+link.rel = 'stylesheet';
+link.href = "https://cdnjs.cloudflare.com/ajax/libs/balloon-css/0.5.0/balloon.min.css";
+
+head.appendChild(link);
+body.appendChild(fileIssueLink);
+
+body.onmouseup = function () {
+  var textSelected = window.getSelection().toString();
+  var newIssueURL = new URL('new?title=&body=',issueBase);
+  var searchParams = newIssueURL.searchParams;
+
+  if (textSelected) {
+    searchParams.set('title',`'${textSelected}'`);
+    searchParams.set('body',textSelected);
+
+    fileIssueLink.style.display = 'inherit';
```

Comment:
would prefer here, and below, we just set a CSS class instead (i.e., don't set `style` directly). I'm still a bit confused tho... why don't we just let balloon.css handle everything? Is that possible... then we don't need to add any CSS and we don't need "#respec-file-issue-btn" at all.

Reply:
yes, this is better. I'll use the inbuilt css of `Balloon.css`